explainer.md

```diff
@@ -53,7 +51,13 @@ See individual sections for more details on these characteristics.
 
 ## Type definitions
 
-The central part of the Typed Objects specification are *type definition objects*, generally called *type definitions* for short. Type definitions describe the fixed structure of a value.
+The central part of the Typed Objects specification are *type definition objects*, generally called *type definitions* for short. Type definitions describe the fixed structure of a value, and are used to specify a struct type's fields' types.
+
+A type definition has an internal method `[[PerformTypeCheck]]`, which, given a value `V`, returns a new value `vResult` which is of the right type, or throws.
+
+The behavior of `[[PerformTypeCheck]]` for all types provided by this specification is described below.
+
+Host environments can provide exotic objects with different implementations of `[[PerformTypeCheck]]`.
 
 ### Value type definitions
```

Comment:
Maybe rename to just "Value Types"?  "Definition" seems like it would be reserved for the compound things like struct/array-types.

Reply:
I went with just "Value Types", hope you agree that that works.

Reply:
That's great, thanks!